docs/cel2/operators/run-node.md

```diff
@@ -121,11 +287,13 @@ The following sections contain all infromation required to set up your node from
   - [op-node](https://us-west1-docker.pkg.dev/devopsre/celo-blockchain-public/op-node:celo-v2.0.0-rc4)
   - [eigenda-proxy](https://ghcr.io/layr-labs/eigenda-proxy:v1.6.4)
 
-### Common problems
+### Troubleshooting
+
+Please reach out to our team on Discord if your problem is not answered below.
```

Comment:
More details: at least a link to the Discord server, better yet mention channel #celo-l2-support (link is https://discord.com/channels/600834479145353243/1286649605798367252, but will not work for people not already in the channel I think)